.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Can you pull the `.idea` folder out please? Seems like a good thing to do in a separate PR

Reply:
Yea that was unintentional, I'll remove those

Reply:
done

Reply:
cool, we can add it back in #3